src/v2/Apps/_About2/AboutApp.tsx

```diff
@@ -269,7 +282,7 @@ const AppDownloadSection: React.FC = () => {
   )
 }
 
-const CollectorInfoSection: React.FC = () => {
+const CollectorInfoComponent: React.FC = () => {
```

Comment:
Just in general, with react stuff, no need to encode the word 'component' in the variable name here and elsewhere, since that's implied by the type -- React.FC (FC = "Function Component")

Reply:
ah shoot I was trying to be uniform with naming, but thats an easy fix thanks for catching that

Reply:
gonna refactor in a future PR and remove the legacy code in the /about page and do any code cleanup addressed here